File: algorandsamples/v2/myjsdemo/indexer/accountInfo.js
// AccountInfo.js
// requires algosdk@1.6.1 or higher 
// verify installed version
// npm list algosdk
const algosdk = require('algosdk');

// const indexer_token = "";
// const indexer_server = "http://localhost";
// const indexer_port = 8980;

// purestake hackathon 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab'

const indexer_server = "https://testnet-algorand.api.purestake.io/idx2/";
const indexer_port = "";
const indexer_token = {
  'X-API-key': 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab',
}

// const indexer_server = "https://indexer-internal-testnet.aws.algodev.network/";
// const indexer_port = "443";
// const indexer_token = "YddOUGbAjHLr1uPZtZwHOvMDmXvR1Zvw1f3Roj2P"
// Instantiate the indexer client wrapper
// let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);
(async () => {

    let acct = "NI2EDLP2KZYH6XYLCEZSI5SSO2TFBYY3ZQ5YQENYAGJFGXN4AFHPTR3LXU";   
    let accountInfo = await indexerClient.lookupAccountByID(acct).do();
    console.log("Information for Account: " + JSON.stringify(accountInfo, undefined, 2));
})().catch(e => {
    console.log(e.message);
    console.trace();
});

File: algorandsamples/v2/mypythondemo/indexer/account_info.py
# account_info.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

# instantiate indexer client
myindexer = indexer.IndexerClient(indexer_token="", indexer_address="http://localhost:8980")
# token = {
#     'X-API-key': 'WpYvadV1w53mSODr6Xrq77tw0ODcgHAx9iJBn5tb',
# }
# myindexer = indexer.IndexerClient(indexer_token=token,
#                                   indexer_address="https://betanet-algorand.api.purestake.io/idx2")

# response = myindexer.account_info(
#     address="7WENHRCKEAZHD37QMB5T7I2KWU7IZGMCC3EVAO7TQADV7V5APXOKUBILCI")

# response = myindexer.account_info(
#     address="ZV2CISJONFBUIUIYFKQJ2UISXPDESCJWH6WO6DKRMXUJB7HDBSABOTYFKU")
# response = myindexer.account_info(
#     address="NI2EDLP2KZYH6XYLCEZSI5SSO2TFBYY3ZQ5YQENYAGJFGXN4AFHPTR3LXU")

response = myindexer.account_info(
    address="7DCJZKC4JDUKM25W7TDJ5XRTWGUTH6DOG5WARVA47DOCXQOTB4GMLNVW7I")

# ZV2CISJONFBUIUIYFKQJ2UISXPDESCJWH6WO6DKRMXUJB7HDBSABOTYFKU

# https://betanet-algorand.api.purestake.io/idx2
# response = myindexer.account_info(
#     address="6SKIRCMLFSSY3EJUC6QGFM3TFIJH72ZYUHX7GCUBDBUBYCAHJBJ5PWB344")

print("Account Info: " + json.dumps(response, indent=2, sort_keys=True))
